valuesets/ValueSet-UKCore-ReportCodeSnCT.xml

```diff
@@ -4,8 +4,8 @@
 	<version value="2.0.0"/>
 	<name value="UKCoreReportCodeSnCT"/>
 	<title value="UK Core Report Code SnCT"/>
-	<status value="draft" />
-	<date value="2021-09-10" />
+	<status value="retired" />
+	<date value="2023-02-14" />
```

Comment:
Do we normally change the date when we retire? If so, probably worth making sure thats documented?